added tracing to load_var and set_pipeline. (#5706)
concourse/concourse: release-notes/latest.md

#### <sub><sup><a name="5222" href="#5222">:link:</a></sup></sub> feature

* Proxy support for NewRelic emitter

> #### <sub><sup><a name="5706" href="#5706">:link:</a></sup></sub> fix

Reviewer: ```suggestion
#### <sub><sup><a name="5706" href="#5706">:link:</a></sup></sub> feature
```

> * Steps `load_var` and `set_pipeline` missed tracing, @evanchaoli added tracing to these steps. #5706

Reviewer: ```suggestion
* @evanchaoli added spans for the `load_var` and `set_pipeline` steps when [distributed tracing](https://concourse-ci.org/tracing.html) is enabled. #5706
```
